tests/test_data.py

```diff
@@ -28,6 +29,16 @@ def test_story_file_with_minimal_story_is_story_file(tmpdir: Path):
     assert is_story_file(str(p))
 
 
+def test_default_conversation_tests_are_conversation_tests(tmpdir: Path):
```

Comment:
let's also cover the case where another file (such as `nlu.md` or `domain.yml`) is in the test directory